packages/components/pie-button/README.md

```diff
@@ -42,165 +28,13 @@ $ npm i @justeattakeaway/pie-button
-
-### Using `pie-icons-webc` with `pie-button` icon slot
-
-We recommend using `pie-icons-webc` when using the `icon` slot. Here is an example of how you would do this:
-
-```html
-<!--
-  Note that pie-button and the icon that you want to use will need to be imported as components into your application.
-  See the `pie-icons-webc` README for more info on importing these icons.
--->
-<pie-button>
-    <icon-plus-circle slot="icon"></icon-plus-circle>
-    Search
-</pie-button>
-```
-
-
-## Events
-
-This component does not use any custom event handlers. In order to add event listening to this component, you can treat it like a native HTML element in your application.
-
-For example, to add a click handler in various templates:
-
-### HTML
-
-```html
-<!-- Other attributes omitted for clarity -->
-<pie-button onclick="e => console.log(e)">Click me!</pie-button>
-```
-
-### Vue templates (using Nuxt 3)
-
-```html
-<!-- Other attributes omitted for clarity -->
-<pie-button @click="handleClick">Click me!</pie-button>
-```
+## Documentation
 
-### React templates (using Next 13)
-
-```html
-<!-- Other attributes omitted for clarity -->
-<PieButton onClick={handleClick}>increment</PieButton>
-
-```
-
-## Forms usage
-
-The `pie-button` web component is designed to integrate with standard HTML forms just like a native HTML button. When positioned inside a form, the component will automatically associate itself, enabling it to directly interact with the form context.
-
-### Button Attributes
-
-The `pie-button` provides a set of attributes to customize its behavior within forms:
-
-- `type`: Determines the button's function. Set to `submit` for form submissions or `reset` to clear form fields.
-- `formaction`: Designates an alternative URL for form data submission when this specific button is clicked.
-- `formenctype`: Specifies the form data encoding type during submission via this button.
-- `formmethod`: Sets the HTTP method (e.g., GET or POST) for form data when initiated by this button.
-- `formnovalidate`: If present, ensures the form is submitted without validation checks.
-- `formtarget`: Dictates where to display the response after form submission.
-
-Please see the [MDN docs]((https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#attributes)) for more information on these attributes.
-
-### Integration Example
-
-```html
-<form action="/default-endpoint" method="post">
-    <input type="text" name="username" required>
-    <pie-button
-        type="submit"
-        formaction="/alternate-endpoint"
-        formenctype="multipart/form-data"
-        formmethod="post"
-        formnovalidate
-        formtarget="_blank">
-        Submit
-    </pie-button>
-</form>
-```
+Visit  [Button | PIE Design System](https://pie.design/components/button/overview) to view more information on this component.
```

Comment:
Can you copy the example in the `pie-switch` component? It uses some nice formatting for GH :)